hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -66,12 +72,20 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setTaskId(nameNodeTask.getTaskInfo().getTaskId())
           .setState(TaskState.TASK_RUNNING)
           .build());
+      healthCheckNN = new TimedHealthCheck(driver, nameNodeTask);
+      timer.scheduleAtFixedRate(healthCheckNN,
+        hdfsFrameworkConfig.getHealthCheckWaitingPeriod(),
```

Comment:
This can be timer.scheduleAtFixedRate(new TimedHealthcheck(driver, nameNodeTask), .... right?